### PR TITLE
[vm] upgrade move coverage tool

### PR DESCRIPTION
**Pull request by guangyuz**

### CODE REVIEW

**Comment by guangyuz** [comment]
Wrong address format (0x0x...) caused by starcoin change of fmt::Display for AccountAddress. The change is applied for specific sdk reason. Currently there is no plan to fix it so we have the work around above.

File: vm/move-coverage/src/coverage_map.rs [hunk 1]
```diff
@@ -86,23 +143,140 @@ impl ModuleCoverageMap {
...
+
 pub fn output_map_to_file<M: Serialize, P: AsRef<Path>>(file_name: P, data: &M) -> Result<()> {
     let bytes = scs::to_bytes(data)?;
     let mut file = File::create(file_name)?;
     file.write_all(&bytes)?;
     Ok(())
 }
+
+pub fn preprocess_address(literal: &str) -> &str {
+    if literal.starts_with("0x0x") {
```
Comment (jolestar) [inline vm/move-coverage/src/coverage_map.rs:277]: 这是哪里序列化 address 的时候手动拼接了一个 0x？

Comment (guangyuz) [reply vm/move-coverage/src/coverage_map.rs:277]: https://github.com/starcoinorg/libra/commit/523b226f0a2b183e59cc6a208449e0c51b269138

Comment (guangyuz) [reply vm/move-coverage/src/coverage_map.rs]: 确认了一下，这个是统一地址格式时有些地方没有完全覆盖到，已经在libra这边处理了。

**Review by jolestar — APPROVED** [review]

**Pull request merged**